Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -665,15 +665,15 @@ void ActivateBulletinBoard(RaycastHit hit, DaggerfallBulletinBoard bulletinBoard
             };
 
             // formatting message is split into 2 parts, depending whether we got any news or not.
-            if (bulletinBoardMessage != string.Empty)
+            if (bulletinBoardMessage != null)
```

Comment:
What about `string.IsNullOrEmpty`?

Reply:
that's no longer a string, that's a TextFile.Token[]

Reply:
I see, I didn't scroll up and didn't think the type could have changed; because of `var`